.env

```diff
@@ -1,7 +1,7 @@
 MINIO_VERSION=latest
 MINIO_UI_PORT=9000
 MINIO_REGION=us-east-1
-MINIO_DOMAIN=candig.local
+MINIO_DOMAIN=host.docker.internal
```

Comment:
changed from candig.local because this is now indirectly given to the workflow

Reply:
This solution only works on Windows and OS X, since host.docker.internal does not work on Linux. This is intended to be solved by implementing Traefik to handle routing

Reply:
This is fine for now, also it is easy for users to change this value for their respective environments